resources/kcp/charts/kyma-metrics-collector/templates/prometheusrule.yaml

```diff
@@ -0,0 +1,111 @@
+{{- if .Values.global.kyma_metrics_collector.enabled -}}
+{{- if .Values.prometheus.rules.enabled -}}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+{{ include "kyma-metrics-collector.prometheusrule.labels" . | indent 4 }}
+  name: {{ include "kyma-metrics-collector.fullname" . }}
+  namespace: {{ .Values.prometheus.namespace }}
+spec:
+  groups:
+    - name: kmc.rules
```

Comment:
(nit): let's use `EDP`, `KMC`, `KEB`, `SKR` consistently all over the places as these acronyms